static/toasting,js

```diff
@@ -0,0 +1,9 @@
+function toasting() {
```

Comment:
This filename `toasting,js` seems to have comma for the extension instead of a period?

Reply:
OK, that one I could not test yet as it is not yet included in the code with a tag